CHANGES.rst

```diff
@@ -31,6 +31,8 @@ associations
 datamodels
 ----------
 
+- Update to exclude target_acquisitions from processing in the calwebb_tso3 pipeline [#3759]
```

Comment:
Because this update was actually done within the `calwebb_tso3` pipeline module, the change log entry should also be in a section labeled as "calwebb_tso3", instead of under "datamodels".